src/protocols/aeternity/views/AccountDetails.vue

```diff
@@ -1,7 +1,10 @@
 <template>
   <IonPage>
     <IonContent class="account-ion-content">
-      <AccountDetailsBase class="account-details">
+      <AccountDetailsBase
+        class="account-details"
+        :ionic-lifecycle-status="ionicLifecycleStatus"
```

Comment:
Instead of passing the prop deeper and deeper just use `v-if` in one of the parents and display/hide the element to force it to render again when the ionic lifecycle changes?